File: autoskola-central-back/src/main/java/hr/autoskola/model/Ispit.java
package hr.autoskola.model;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class Ispit {

	private Integer ispitId;
	private Integer korisnikId;
	private Integer maksimalniBrojBodova;
	private Integer ostvareniBrojBodova;
	private String statusIspita;
	private String creationDate;
	
	private Korisnik korisnik;
	private List<Pitanje> pitanja;
	
}

File: autoskola-central-front/src/app/core/models/ispit.ts
import { Korisnik } from "./korisnik";
import { Pitanje } from "./pitanje";

export class Ispit{

    ispitId : number | undefined;
    korisnikId : number | undefined;
    maksimalniBrojBodova !: number;
    ostvareniBrojBodova !: number ;
    statusIspita !: string;
    creationDate !: string;

    pitanja : Pitanje[] | undefined;
    korisnik !: Korisnik;
}